redirect/variables.tf

```diff
@@ -0,0 +1,59 @@
+variable "create" {
+  description = "Should resources be created"
+  default     = true
+  type        = bool
+}
+
+variable "project" {
+  description = "Kebab-cased name of the project. Will be included in resource names"
+  type        = string
+}
+
+variable "environment" {
+  description = "Kebab-cased name of the environment, eg. production, staging, development, preview. Will be included in resource names"
+  type        = string
+}
+
+variable "name_prefix" {
+  description = "Kebab-cased resource name name prefix, defaults to project-environment"
+  type        = string
+  default     = null
+}
+
+variable "tags" {
+  description = "Additional tags to add to each resource that supports them"
+  type        = map(string)
+  default     = {}
+}
+
+variable "domains" {
+  description = "List of domains which will be redirected. If empty, will use the default cloudfront domain"
+  type        = list(string)
+  default     = []
```

Comment:
I think this should be required, there's no reason to use this module with a default, random cloudfront domain

Reply:
Then the certificate should also be required, moreover the example will be much more harder to implement and understand (?).

Reply:
That's true but we probably shouldn't design the module around the example. I'd prefer the example to match how the module might be actually used. You can check how the `ssl/acm` module example is implemented for reference.

Reply:
👍